src/REAXFF/fix_reaxff_species.h

```diff
@@ -62,7 +62,10 @@ class FixReaxFFSpecies : public Fix {
   int delete_Nsteps, *delete_Tcount;
   double massmin, massmax;
   int singlepos_opened, multipos_opened, del_opened;
-  char *ele, **eletype, *filepos, *filedel;
+  char *filepos, *filedel;
+  std::vector<int> ele2uele;
```

Comment:
Can you add a trailing comment for this vector?
I.e. what is the Ith quantity stored in ele2uele[I] an index into?
In the cpp file it appears both of the 2 next vectors are ntypes in length, so it is not clear.

Reply:
Trailing comments for the other 2 vectors would be good as well since it is not clear how they are different (as in previous comment).  Maybe ueletype is actually of only length nutype?

Reply:
yes, ueletype should only have 'nutypes' values. I think I initialized it to length ntypes to do so at the same place as the other vectors, at which point nutypes is not known yet (noting that nutypes is always <= ntypes)